docs/starting-from-scratch.md

```diff
@@ -10,323 +10,277 @@ single-spa allows you to build micro frontends that coexist and can each be writ
+1. home: a React app using [React Router](https://reacttraining.com/react-router/)
+2. navBar: a React app that always displays top-level navigation
+3. angularJS: an AngularJS app using [angular-ui-router](https://ui-router.github.io/ng1/)
 
-Be sure to read through the [single-spa docs](https://single-spa.js.org/), check the [single-spa github](https://github.com/CanopyTax/single-spa) and the [help section](https://single-spa.js.org/help.html) for more support.
+The complete code for this example is in the [`single-spa-simple-example`](https://github.com/alocke12992/single-spa-simple-example) repository.
 
-## Step One: Project Set up
+> __Note__
+>
+> We encourage you to read through all the [single-spa docs](/) to become familiar with the entire Single-spa setup. Visit the [single-spa Github](https://github.com/CanopyTax/single-spa), the [help section](https://single-spa.js.org/help.html), or our community [Slack](https://join.slack.com/t/single-spa/shared_invite/enQtMzIwMTcxNTU3ODQyLTM1Y2U1OWMzNTNjOWYyZDBlMDJhN2VkYzk3MDI2NzQ2Nzg0MzMzNjVhNWE2YjVhMTcxNjFkOWYzMjllMmUxMjk) channel for more support.
 
-Create a new directory to house our project, followed by a `src` folder to hold all of our micro-service applications. Then, the root of our new project, initialize the package manager of your choice and install single-spa. For this tutorial, we will be using [yarn](https://yarnpkg.com/en/).
+## 1. Initial setup
 
-```bash
+> __Note__
+>
+> For this tutorial, we will be using [yarn](https://yarnpkg.com/en/) but npm has its own equivalent commands and can be used almost interchangibly.
+
+Create a new folder for this project and navigate into it. Initialize a new project using your package manager, and then install single-spa as a dependency. Then create a *src* folder to hold all of our micro-service applications, with each in their own folder.
+
+```sh
 mkdir single-spa-simple-example && cd single-spa-simple-example
-mkdir src
 yarn init              # or npm init
 yarn add single-spa    # or npm install --save single-spa
+mkdir src
 ```
 
-### a) Setup Babel
+### 1.a Setup Babel
 
-We will be using [Babel](https://babeljs.io/) to compile our code. Run the following command to install the necessary dependencies:
+We will be using [Babel](https://babeljs.io/) to compile our code. Install it and some additional dependencies using:
 
-```bash
-yarn add babel-core babel-plugin-syntax-dynamic-import babel-plugin-transform-object-rest-spread babel-preset-env babel-preset-latest babel-preset-react --dev
+```sh
+yarn add babel-core babel-preset-env babel-preset-latest babel-preset-react babel-plugin-syntax-dynamic-import babel-plugin-transform-object-rest-spread --dev
```

Comment:
It would be nice to update this to babel 7:

```sh
yarn add --dev babel-core@7.0.0-bridge.0 @babel/preset-env @babel/preset-react @babel/plugin-syntax-dynamic-import @babel/plugin-proposal-object-rest-spread
```

Reply:
Might be nice to PR the code repo for this, too. https://github.com/alocke12992/single-spa-simple-example

Reply:
Would this be better treated as a separate PR? Or do you really want it included with these changes? I'd be fine either way.

Reply:
separate PR is fine 👍